CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning
     moveit_ros_planning_interface
     moveit_ros_perception
+    rviz
```

Comment:
Also needs to be added to `CATKIN_DEPENDS` args in `catkin_package`, below, to make `catkin_lint` happy.